src/wallet/wallet.cpp

```diff
@@ -1200,7 +1200,7 @@ CPubKey CWallet::GenerateNewHDMasterKey()
 
         // write the key&metadata to the database
         if (!AddKeyPubKey(key, pubkey))
-            throw std::runtime_error("CWallet::GenerateNewKey(): AddKey failed");
+            throw std::runtime_error("CWallet::GenerateNewHDMasterKey(): AddKey failed");
```

Comment:
The goal was to keep this in case some refactoring makes it possible that it fails.